verticapy/learn/tools.py

```diff
@@ -0,0 +1,611 @@
+# (c) Copyright [2018-2021] Micro Focus or one of its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# |_     |~) _  _| _  /~\    _ |.
+# |_)\/  |_)(_|(_||   \_/|_|(_|||
+#    /
+#              ____________       ______
+#             / __        `\     /     /
+#            |  \/         /    /     /
+#            |______      /    /     /
+#                   |____/    /     /
+#          _____________     /     /
+#          \           /    /     /
+#           \         /    /     /
+#            \_______/    /     /
+#             ______     /     /
+#             \    /    /     /
+#              \  /    /     /
+#               \/    /     /
+#                    /     /
+#                   /     /
+#                   \    /
+#                    \  /
+#                     \/
+#                    _
+# \  / _  __|_. _ _ |_)
+#  \/ (/_|  | |(_(_|| \/
+#                     /
+# VerticaPy is a Python library with scikit-like functionality to use to conduct
+# data science projects on data stored in Vertica, taking advantage Vertica’s
+# speed and built-in analytics and machine learning features. It supports the
+# entire data science life cycle, uses a ‘pipeline’ mechanism to sequentialize
+# data transformation operations, and offers beautiful graphical options.
+#
+# VerticaPy aims to solve all of these problems. The idea is simple: instead
+# of moving data around for processing, VerticaPy brings the logic to the data.
+#
+#
+# Modules
+#
+# VerticaPy Modules
+from verticapy.toolbox import *
+
+#
+# ---#
+def check_model(name: str, cursor=None):
```

Comment:
I think it would be much nicer if you rename this function to `does_model_exist`, and return a Boolean. In other words, returning true when the model already exists, and false otherwise.
IMHO, using raised errors instead of conditional statements makes the code less readable and its debugging more difficult. This will lead to more challenging code maintenance and less community contribution.

Reply:
Or, you may make the `check_model` function return an integer: 0 if the model doesn't exist, 1 if it is of type1 (native), and 2 if it's of type2 (non-native).

Reply:
Thanks for the idea! It is done and it will be pushed after the v0.5.0 release.

Reply:
Done;